src/Histogram.ts

```diff
@@ -144,21 +146,63 @@ export default class Histogram {
       }
     }
 
+    // Edge case: b and e are both out of bounds 
+    if (b < 0 && e < 0) {
+      return {
+        lut: lut,
+        controlPoints: [
+          { x: 0, opacity: 1, color: [255, 255, 255] },
+          { x: 255, opacity: 1, color: [255, 255, 255] }
+        ],
+      };
+    }
+    if (b > 255 && e > 255) {
+      return {
+        lut: lut,
+        controlPoints: [
+          { x: 0, opacity: 0, color: [255, 255, 255] },
+          { x: 255, opacity: 0, color: [255, 255, 255] }
+        ],
+      };
+    }
+
+    // Generate 2 to 4 control points for a minMax LUT, from left to right
     const controlPoints: ControlPoint[] = [];
+
+    // Add starting point at x = 0
     let startVal = 0;
-    if (b <= 0) {
-      startVal = -b / (e - b);
+    if (b < 0) {
+      if (e === b) {
```

Comment:
I think your handling of edge case e<0 && b<0 (line 150) makes line 175 unreachable.  I'll leave it up to you if you'd prefer to keep it here but it could be removed.

Reply:
Good point, removed

Reply:
This made me think this change (https://github.com/AllenInstitute/volume-viewer/pull/33/commits/a59293b66be7da9f669050dd8b260c37df27b4f1) would also be good, but let me know if that was overkill / more confusing!

Reply:
looks good to me!  if b>=255 then the whole range has to be zero opacity (assuming e>=b which it must be!).